packages/vendor-core/lib/modules.js

```diff
@@ -12,6 +12,7 @@ module.exports = [
   'core-js/shim',
   'regenerator-runtime/runtime',
   'formik',
+  'rc-number-input',
```

Comment:
Should be `rc-input-number`, looks like the build if failing due to this reason, see:
https://github.com/theforeman/foreman-js/pull/198/checks?check_run_id=992854785#step:6:193